### PR TITLE
Create submit form route

### PR DESCRIPTION
**Pull request by edgarfinn**

### CODE REVIEW

File: src/handlebars/views/report.hbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!-- FORM -->
+<div class="form-wrap">
+
+  <form class="report-form" action="/" method="post">
+
+    <button class="form-button" id="getLocationButton" type="button" name="location">Submit Location</button>
```
Comment (jwld) [inline src/handlebars/views/report.hbs]: probs best to stick to one naming convention - I think we've gone for kebab case in most places, so maybe change to get-location-button

File: public/script/report.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var locationButton = document.getElementById('getLocationButton');
+
+function getLocation() {
+    if (navigator.geolocation) {
```
Comment (jwld) [inline public/script/report.js]: indentation ✌️

File: public/script/report.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+var locationButton = document.getElementById('getLocationButton');
```
Comment (jwld) [inline public/script/report.js]: might be good to wrap all this code in an IIFE

File: public/script/report.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+function getLocation() {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(showPosition);
+    } else {
+        x.innerHTML = "Geolocation is not supported by this browser.";
+    }
+}
```
Comment (jwld) [inline public/script/report.js]: missing semicolon - also on lines 15, 16, 18

File: public/script/report.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+function showPosition(position) {
+    console.log(
+    "Latitude: " + position.coords.latitude +
+    "Longitude: " + position.coords.longitude
+  )
+}
+
+locationButton.addEventListener('click', getLocation)
```
Comment (jwld) [inline public/script/report.js]: why separate getLocation out? you could even get away with not having any variables, e.g.
```
document.getElementById('getLocationButton').addEventListener('click', function() {
    ...
});
```

**Review by jwld — CHANGES_REQUESTED** [review]
just a few suggestions in report.js and report.hbs

File: public/script/report.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+var locationButton = document.getElementById('getLocationButton');
+
+function getLocation() {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(showPosition);
+    } else {
+        x.innerHTML = "Geolocation is not supported by this browser.";
```
Comment (jwld) [inline public/script/report.js]: what's x?

File: public/script/report.js (hunk 3 above)
Comment (jwld) [reply public/script/report.js]: by all this code I mean the whole file

File: public/style/sass/_report.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  outline:none;
+  width: 100%;
+  height: 40px;
+  padding: 10px;
+  border: 1px solid #ccc;
+  background-color: #eee;
+  font: 15px Helvetica;
+  cursor: pointer;
+  background-color: lightgrey;
+  padding: 0.5em;
```
Comment (jwld) [inline public/style/sass/_report.scss]: got padding twice here - the second overwrites the first (line 22)

Comment (jwld) [reply public/style/sass/_report.scss]: same with background color

File: src/handlebars/views/report.hbs (hunk 1 above)
Comment (edgarfinn) [reply src/handlebars/views/report.hbs]: Classes: kebab. Id's: Camel

File: public/script/report.js (hunk 2 above)
Comment (edgarfinn) [reply public/script/report.js]: done

File: public/script/report.js (hunk 4 above)
Comment (edgarfinn) [reply public/script/report.js]: done. 

And for now just logging out the error message

**Pull request merged**